presto-product-tests/conf/tempto/tempto-configuration-for-docker-kerberos.yaml

```diff
@@ -9,18 +9,27 @@ databases:
   hive:
     host: hadoop-master
     jdbc_url: jdbc:hive2://${databases.hive.host}:10000/default;principal=hive/hadoop-master@LABS.TERADATA.COM;auth=kerberos;kerberosAuthType=fromSubject;
-    jdbc_user: hive/hadoop-master@LABS.TERADATA.COM
+    jdbc_user: random_string # jdbc_user is not being used in kerberized environment
```

Comment:
Or we need it for cli authorization checks in which case is the value appropriate